semcore/tooltip/src/index.d.ts

```diff
@@ -15,6 +20,16 @@ export type TooltipProps = PopperProps &
      * @default default
      */
     theme?: 'default' | 'warning' | 'invert';
+
+    /**
+     * @deprecated Tooltip should have only one interaction - `hover`. You shouldn't use it with another interactions.
```

Comment:
Let's just say "Only `hover` interaction allowed for Tooltip"

Reply:
I thought, that we will remove `interaction` prop from Tooltip at all...

Reply:
Ok, lets just remove the deprecation description. It's more confusing than informative

Reply:
But this is deprecated prop... We woun't need it in the next major release

Reply:
So, let's just say only about removing in the next major release

Reply:
I removed `@deprecated` and add a comment. But I disagree =)